job-dsls/jobs/reduced_drools_release.groovy

```diff
@@ -0,0 +1,241 @@
+/**
+ * Job that runs the community release pipeline, but only over the reps until & incl. droolsjbpm-integration
+ */
+
+import org.kie.jenkins.jobdsl.Constants
+
+def kieVersion=Constants.KIE_PREFIX
+def baseBranch=Constants.BRANCH
+def releaseBranch="r7.45.0.t20201015"
+def organization=Constants.GITHUB_ORG_UNIT
+def m2Dir = Constants.LOCAL_MVN_REP
+def commitMsg="Upgraded version to "
+
+
+// creation of folder
+folder("reduced-drools-release")
+
+def folderPath="reduced-drools-release"
+
+def redRelease='''
+pipeline {
+    agent {
+        label 'kie-rhel7 && kie-mem24g'
+    }
+    tools {
+        maven 'kie-maven-3.6.3'
+        jdk 'kie-jdk1.8'
+    }
+    stages {
+        stage('CleanWorkspace') {
+            steps {
+                cleanWs()
+            }
+        }
+        stage('Checkout droolsjbpm-build-bootstrap') {
+            steps {
+                checkout([$class: 'GitSCM', branches: [[name: '$baseBranch']], browser: [$class: 'GithubWeb', repoUrl: 'git@github.com:$organization/droolsjbpm-build-bootstrap.git'], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'droolsjbpm-build-bootstrap']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'kie-ci-user-key', url: 'git@github.com:$organization/droolsjbpm-build-bootstrap.git']]])
+                dir("${WORKSPACE}" + '/droolsjbpm-build-bootstrap') {
+                    sh  'pwd \\n' +
+                        'git branch \\n' +
+                        'git checkout -b $baseBranch'
+                }
+            }    
+        }
+        stage ('Replace repository-list.txt and change branched-7-repository-list.txt') {
+            steps {
+                configFileProvider([configFile(fileId: '1a43573a-318c-426a-bb2b-c9df7fe97a02', targetLocation: 'repository-list.txt', variable: 'REP_LIST')]) {
+                    dir("${WORKSPACE}") {
+                        sh 'cp repository-list.txt droolsjbpm-build-bootstrap/script/repository-list.txt \\n' +
```

Comment:
Where does this repository-list.txt file come from?

Reply:
so answering my own question, there is managed file on Jenkins with reduced content